apps/web2/src/pages/FriendsPage/FriendsPage.tsx

```diff
@@ -314,16 +340,36 @@ export default function FriendsPage(): JSX.Element {
                 </div>
                 {/* End line breaker */}
                 <br></br>
+                {/* Negative Alert Banner */}
+                {displayError.length > 0 && (
+                  <div className="rounded-md mb-4 bg-red-50 p-4 sm:mx-auto">
+                    <div className="flex">
+                      <div className="flex-shrink-0">
+                        <XCircleIcon
+                          className="h-5 w-5 text-red-400"
+                          aria-hidden="true"
+                        />
+                      </div>
+                      <div className="ml-3">
+                        <h3 className="text-sm font-medium text-red-800">
+                          Error: Invalid Email Address
+                        </h3>
+                        <div className="mt-2 text-sm text-red-700">
+                          {displayError}
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                )}
                 <form onSubmit={onAddFriendHandler}>
                   <label htmlFor="email-addres" className="sr-only">
                     Email address
                   </label>
                   <input
                     id="email-address"
                     name="email"
-                    type="email"
+                    type="text"
```

Comment:
Believe that `type="email"` is correct here.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/email

Reply:
Making it `type="text"` allows us to accept any string input and rely on the regex test to validate the string as an email. Helps make errors uniform.

-- Marmar